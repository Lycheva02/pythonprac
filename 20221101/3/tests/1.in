f = Grange(2, 3, 57)
print(len(f))
t = Grange(10792, 17, 10790)
print(bool(f), bool(t))
print(t[6])
print(f[13:1076], t[36:893:2], sep = "$")
print(f)
print(*list(t[3:742]))