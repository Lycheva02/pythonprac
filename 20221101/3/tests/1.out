4
True False
260492644648
grange(13, 3, 1076)$grange(36, 289, 893)
grange(2, 3, 57)
3 51
